### PR TITLE
Grammar exrecises from lecture 1 have been done.

### PR DESCRIPTION
**Pull request by usernaft**

### CODE REVIEW

File: homework/english/viacheslav_stashchuk/lecture1/task1_grammar.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+Your mum (work) today? - Is your mum working today? 
+Why this boys (to talk) now? - Why are these boys talking now?
+Where he (to live)? - Where does he live?
+Their mother (cook) very well? - Is their mother cooking very well?
```
Comment (olgaz-skiba) [inline homework/english/viacheslav_stashchuk/lecture1/task1_grammar.md:5]: Does their mother cook very well?

File: homework/english/viacheslav_stashchuk/lecture1/task1_grammar.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Mike (not wear) a tie today. - Mike is not wearing a tie today.
+Maila (not to watch) TV every morning. - Maila does not watch TV every morning.
+Listen! Blanca (sing) in the bathroom. - Listen! Blanca is singing in the bathroom.
+Bea sometimes (read) comics. - Bea sometimes reads comics.
+
+#Найдите ошибки и исправьте их.
+We not going to school today. - We are not going to school today.
+What you doing after school? - What are you doing after school?
+At the moment Peter is work in Russia. - At the moment Peter is working in Russia.
+Does he got a new car? - Has he got a new car?
```
Comment (olgaz-skiba) [inline homework/english/viacheslav_stashchuk/lecture1/task1_grammar.md:31]: this is also correct, but for the past simple
Does he **get** a new car?

**Review by olgaz-skiba — DISMISSED** [review]
lgfm

File: homework/english/viacheslav_stashchuk/lecture1/task1_grammar.md (hunk 2 above)
Comment (usernaft) [reply homework/english/viacheslav_stashchuk/lecture1/task1_grammar.md:31]: It is correct. When we are talking about possession, relationships, illnesses and characteristics of people or things we can use either have or have got. The have got forms are more common in an informal style. Have got has the same meaning as have and both are used as present tenses. Note that have got is not the present perfect of get

**Review by olgaz-skiba — APPROVED** [review]
lgfm

**Review by ElenaOwa — APPROVED** [review]

**Review by youRaFyodorov — APPROVED** [review]
lgfm

**Pull request merged**